sql/encryption-and-compression-functions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 加密和压缩函数
+category: user guide
+---
+
+# 加密和压缩函数
+
+
+| 函数名                                                                                                                                               | 功能描述                                       |
+|:------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
```

Comment:
use width ratio of table body column.

for long tables, no need to align with spaces.

like:

```
| 函数名 | 功能描述 |
|:--------------|:--------------------------------------------------|
```